proxy/src/main/java/net/md_5/bungee/connection/InitialHandler.java

```diff
@@ -353,6 +353,12 @@ public void handle(LoginRequest loginRequest) throws Exception
     {
         Preconditions.checkState( thisState == State.USERNAME, "Not expecting USERNAME" );
 
+        if ( loginRequest.getData().isEmpty() )
+        {
+            disconnect( bungee.getTranslation( "name_empty" ) );
```

Comment:
Why does it need its own message

Reply:
I thought it would be more acurat if it has the message we could send the invalid chars messages but it actually has no invalid chara